leaders/src/main/java/edu/cornell/resultsmanager/TestOutputSender.java

```diff
@@ -0,0 +1,45 @@
+package edu.cornell.resultsmanager;
+
+import lombok.extern.slf4j.Slf4j;
+
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStreamReader;
+import java.io.OutputStream;
+import java.net.HttpURLConnection;
+import java.net.MalformedURLException;
+import java.net.URL;
+
+@Slf4j
+public class TestOutputSender {
+
+    public static void sendResults(String json, String url) {
+        try {
+            URL postUrl = new URL(url);
```

Comment:
Getting a warning that the `URL()` constructor is deprecated:

Deprecated. Use `URI.toURL()` to construct an instance of URL.